test/test_autograd.py

```diff
@@ -1379,6 +1379,7 @@ def test_unused_output(self):
         expected_grad[:2] = grad_output
         self.assertEqual(x.grad.data, expected_grad)
 
+    @skipIfRocm
```

Comment:
This one is really weird, if we adjust the input_length at https://github.com/pytorch/pytorch/blob/e734c94fa2d4399d33b37a3dcc9096a0dbb776e5/test/test_autograd.py#L1392 from 150 to be <= 138 then the test can pass, any value > 138 fails.

The failure was introduced in https://github.com/pytorch/pytorch/commit/a7e3cd09e023685a96f65d8447490455a16a4ebb.